arch/arm64/configs/enchilada_defconfig

```diff
@@ -742,18 +742,67 @@ CONFIG_AUTOFS4_FS=y
 
 CONFIG_LOCALVERSION="-hybris"
 
-# Recommended
+# Memory
 CONFIG_MEMCG=y
 CONFIG_MEMCG_KMEM=y
 CONFIG_MEMCG_SWAP=y
 
-
-# telnet
-
+# Telnet
 CONFIG_USB_F_RNDIS=y
 CONFIG_USB_CONFIGFS_RNDIS=y
 CONFIG_USB_F_GSI=n
 CONFIG_USB_CONFIGFS_F_GSI=n
 CONFIG_USB_CONFIGFS_F_QCGSI=n
 CONFIG_USB_CONFIGFS_QCRNDIS=n
-CONFIG_USB_CONFIGFS_RMNET_BAM=n 
\ No newline at end of file
+CONFIG_USB_CONFIGFS_RMNET_BAM=n
+
+# EXT4
+CONFIG_LBDAF=y # suport large ext4
+
+# CIFS
```

Comment:
Don't think there is enough demand for CIFS or NFS to add all this to the defconfig.